src/main/java/io/confluent/copycat/hdfs/HdfsWriter.java

```diff
@@ -61,6 +70,9 @@
   private long backOffMs;
   private Set<TopicPartition> lastAssignment;
   private Map<TopicPartition, Long> failureTime;
+  private Map<TopicPartition, Schema> schemas;
+  private Map<TopicPartition, Integer> versions;
```

Comment:
Heh, the number of maps is really getting out of control here. We should definitely follow up sometime soon with a cleanup of these.